observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-about"]
+= Logging 6.0
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` custom resource (CR) is the central configuration point for log collection and forwarding.
+
+
+
+
+
+== Inputs and Outputs
+
+Inputs specify the sources of logs to be forwarded. Logging provides built-in input types: `application`, `infrastructure`, and `audit`, which select logs from different parts of your cluster. You can also define custom inputs based on namespaces or pod labels to fine-tune log selection.
+
+Outputs define the destinations where logs are sent. Each output type has its own set of configuration options, allowing you to customize the behavior and authentication settings.
+
+== Pipelines and Filters
+
+Pipelines determine the flow of logs from inputs to outputs. A pipeline consists of one or more input refs, output refs, and optional filter refs. Filters can be used to transform or drop log messages within a pipeline. The order of filters matters, as they are applied sequentially, and earlier filters can prevent log messages from reaching later stages.
+
+== Operator Behavior
+
+The Cluster Logging Operator manages the deployment and configuration of the collector based on the `ManagementState` field:
+
+- When set to `Managed` (default), the operator actively manages the logging resources to match the configuration defined in the spec.
+- When set to `UnManaged`, the operator does not take any action, allowing you to manually manage the logging components.
+
+
+== Validation
+Logging includes extensive validation rules and default values to ensure a smooth and error-free configuration experience. The `ClusterLogForwarder` resource enforces validation checks on required fields, dependencies between fields, and the format of input values. Default values are provided for certain fields, reducing the need for explicit configuration in common scenarios.
+
+include::modules/log6x-oc-explain.adoc[leveloffset=+1]
```

Comment:
In my view the "About Logging" page is about explaining the concepts of Logging and maybe also providing instructions for a "minimal installation" (or linking to the relevant parts). I don't know how `oc explain` fits into this... I think I would expect this to be in a troubleshooting section or a "further reading" one but not at the end of the Logging introduction.